docs/content/getting-started/setup.md

```diff
@@ -155,6 +155,10 @@ then it **is** supported too and you can [follow the steps for compatible distri
 In case your distribution in not compatible with [get.docker.com](https://get.docker.com), you will need to install 
 latest stable Docker for your distribution first, and then [follow the steps for compatible distributions](#install-linux-debian-fedora).
 
+{{% notice note %}}
```

Comment:
I would combine this paragraph with the one above it, to outline steps necessary on distributions not supported by by the get.docker.com script.

Reply:
updated